### PR TITLE
Add Sonobi Adapter

### PR DESCRIPTION
**Pull request by Studnicky**
Sonobi Adapter for prebid.js

### CODE REVIEW

**Comment by protonate** [comment]
Great, thanks, confirmed the adapter is working as expected, reviewed.

File: integrationExamples/gpt/pbjs_example_gpt.html [hunk 1]
```diff
@@ -117,6 +117,20 @@
                         }
                     },
                     {
+                        bidder: 'sonobi',       //  New format
+                        params: {
+                            dom_id: 'PER SLOT', //  <String> dom Id
```
Comment (CarsonBanov) [inline integrationExamples/gpt/pbjs_example_gpt.html]: @Studnicky Is there a chance you could provide an example of what these parameters would look like?  For example, is `dom_id` something like `ABC124` or `leaderboard `?

**Comment by mkendall07** [comment]
@CarsonBanov 
Please see here: http://prebid.org/dev-docs/bidders.html#sonobi

File: integrationExamples/gpt/pbjs_example_gpt.html (hunk 1 above)
Comment (Studnicky) [inline integrationExamples/gpt/pbjs_example_gpt.html]: DOM stands for "document object model" - the id is the corresponding ID of the ad slot you are targeting.

The id can literally be anything, so long as it is unique to the ad slot.  Some publishers make up their own ID's for their own purposes, others use the div ID's provided by DFP.

The DFP id's often follow the pattern "div-gpt-ad-123456789-1", as you might see here:
`<div id="div-gpt-ad-123456789-1">
<script type="text/javascript">
googletag.cmd.push(function() { googletag.display("div-gpt-ad-123456789-1"); });
</script>
</div>`

Sonobi uses the ID provided by you to differentiate specific ad slots on your page, as we have noted that many publishers choose to use the same ad unit code for multiple ad slots.

**Comment by Studnicky** [comment]
I get a lot of these messages, @mkendall07 - possibly better to just answer it here.

**Comment by CarsonBanov** [comment]
I think perhaps folks are getting stuck here:
https://github.com/prebid/Prebid.js/releases/tag/0.8.0
which is less specific than 
http://prebid.org/dev-docs/bidders.html#sonobi

**Comment by Studnicky** [comment]
I hope the note above answered your questions.  If you run into further issues, please reach out to your account manager at Sonobi.  They can provide a much more comprehensive internal implementation guide.

**Comment by mkendall07** [comment]
@Studnicky 
Thanks for responding. Any suggestions you have to make the dev docs more helpful please submit them as a PR here: https://github.com/prebid/prebid.github.io

**Pull request merged**